### ISSUE TITLE
Kerning AWA, AVA, Wa, Va, etc. (Ẃ, Ŵ, á, ą, ...)

### ISSUE THREAD

**Issue by dawidd4** [Instrument/instrument-serif#4]
The spacing between the letters is fine for me, but I was wondering if it can be lowered even more, like in the fonts "ITC Garamond Condensed Light", "Berthold Garamond Condensed" and "Garamond Nova Condensed Light".
![Instrument Serif comparison](https://github.com/Instrument/instrument-serif/assets/40753083/5d1329b4-f98c-4f2e-9ca9-5bf5a321836a)

**Comment by rfuenzalida** [comment]
Thanks for reaching out! I think there's something in the software you're using that's not taking the kerning info into account, attached I'm sending you a correct working version of the font so you can see the proper spacing.

![Screen Shot 2023-06-13 at 12 58 20](https://github.com/Instrument/instrument-serif/assets/8063083/01ba154b-0b7e-48a7-90c2-9d6c614287cf)

**Comment by dawidd4** [comment]
You're right!! I glued the words together in MS Paint for the rest of the fonts. It's MS Paint that creates the uneven spacing. I think it has something to to with the rasterization - all the glyphs always align to the pixel raster.
